doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -528,167 +534,107 @@ type: Opaque
 EOF
 ```
 
-Next, generate an `AuthPolicy` that uses secrets in your cluster as API keys as follows:
-
-```bash
-cat $oasPath | envsubst | kuadrantctl generate kuadrant authpolicy --oas -
-```
-
-From this, you can see an `AuthPolicy` generated based on your OAS that will look for API keys in secrets labeled `api_key` and look for that key in the header `api_key`. You can now apply this to the Gateway as follows:
-
-```bash
-cat $oasPath | envsubst | kuadrantctl generate kuadrant authpolicy --oas -  | kubectl apply -f -
-```
-
-You should get a `200` from the following `GET` because it has no auth requirement:
-
-```bash
-curl -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
-```
-
-You should get a `401` for the following `POST` request because it does not have any auth requirements:
-
-```bash
-curl -XPOST -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
-```
-
-Finally, if you add your API key header, with a valid key as follows, you should get a `200` response:
-
-```bash
-curl -XPOST -H 'api_key: secret' -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
-```
-
-### Optional: Step 6 - Set up OpenID Connect auth flow (skip if using API key only)
```

Comment:
I removed the oidc in favour of apikey.i think an walkthrough with one way of doing it is easier to follow but happy to add it back if other people think its useful